src/nvim/eval/typval.c

```diff
@@ -3208,9 +3208,36 @@ void tv_free(typval_T *tv)
   }
 }
 
+/// Increment reference count of a typval. Does nothing if typval is not refcounted.
+void tv_increment_refcount(const typval_T *const tv)
```

Comment:
"inc" is the convention found in a few places (inc_cursor, dec_cursor).

```suggestion
void tv_inc_refcount(const typval_T *const tv)
```